bin/e2e-test-runner

```diff
@@ -103,4 +102,6 @@ $SKIP_SETUP_DEV || setupDev
 # Only start webpack if not already running
 nc -vz 127.0.0.1 8234 2> /dev/null || ./bin/start-frontend &
 pnpm dlx cypress open --config-file cypress.e2e.config.ts &
+uv pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
I think the point of `uv` is to be [global](https://github.com/astral-sh/uv#python-discovery), like ppm/npm or other package managers, so that's why we don't include it in the dependency list.